frameworks/cassandra/build.sh

```diff
@@ -9,7 +39,12 @@ ROOT_DIR="$(dirname "$(dirname $FRAMEWORK_DIR)")"
 export TOOLS_DIR=${ROOT_DIR}/tools
 BUILD_DIR=$FRAMEWORK_DIR/build/distributions
 PUBLISH_STEP=${1-none}
-${ROOT_DIR}/gradlew distZip -p ${ROOT_DIR}/sdk/executor
+
+if [ x"$cli_only" = xtrue ]; then
```

Comment:
Should have pointed out this was a style suggestion with no meat to it, but I see now there can be a lot more behind it, so do ignore and carry on to get your actual work done